test/integration/coscheduling_test.go

```diff
@@ -212,23 +320,43 @@ func TestCoschedulingPlugin(t *testing.T) {
 			t.Logf("Start-coscheduling-test %v", tt.name)
 			testutils.CleanupPods(cs, t, pods)
 			pods = make([]*v1.Pod, 0)
+			// create pod group
+			for _, podInfo := range tt.pods {
+				if podInfo.minAvailable == "" {
+					continue
+				}
+				min, err := strconv.Atoi(podInfo.minAvailable)
+				if err != nil {
+					t.Fatal(err)
+				}
+				pg := &v1alpha1.PodGroup{
+					ObjectMeta: metav1.ObjectMeta{Name: podInfo.podGroupName, Namespace: ns.Name},
+					Spec:       v1alpha1.PodGroupSpec{MinMember: int32(min)},
+				}
+				pg, err = extClient.SchedulingV1alpha1().PodGroups(ns.Name).Create(ctx, pg, metav1.CreateOptions{})
```

Comment:
To make each subtest stateless, I'd suggest to `create(pgs)` and `defer cleanup(pgs)` in each subtest. And btw: current logic should go straight to create PG instead of getting MinMember from podInfo.